src/schema/posts.ts

```diff
@@ -173,12 +177,12 @@ export interface SubmitExternalLinkArgs extends ExternalLink {
 export const getPostNotification = async (
   con: DataSource,
   postId: string,
-): Promise<GQLPostNotification> => {
+): Promise<GQLPostNotification | undefined> => {
   const post = await con
     .getRepository(Post)
     .findOne({ where: { id: postId }, select: ['id', 'upvotes', 'comments'] });
   if (!post) {
-    return null;
+    return undefined;
```

Comment:
Probably fine, but keep a close eye on prod if it doesn't break anything accidental (by expecting nulls or something like that.

Reply:
its used for https://github.com/dailydotdev/daily-api/pull/2094#discussion_r1701363975 thats why I changed it